### DISCUSSION TITLE
Determine page break based on html content height

### DISCUSSION THREAD

**Discussion by blpraveen** [spatie/laravel-pdf#53] [Q&A]
Can I know the html content height crossed pdf page height and insert page break.

Examaple:
I have some content it can span to any height and then list of videos to be inserted which will span across the page.

**Comment by ArielMejiaDev** [comment]
@blpraveen  In my case for reports in tables, I want to avoid a table row that starts at the end of a page and continue in the next page.

In this case I set my layout using tailwind and check how many rows looks fine in a single page, eg: 10... so send my collection in chunks of 10 and loop around the table component and pass the params for that 10 users then apply the directive for pageBreak....
then continue with the next 10 rows in another table.

In your case set a default size for the video containers and try manually checking how many videos looks fine for a specific page size would tell how many videos could you add in every chunk and apply the same pattern
